### DISCUSSION TITLE
How to set up proton mail with vaultwarden?

### DISCUSSION THREAD

**Discussion by SpiderUnderUrBed** [dani-garcia/vaultwarden#4683] [Q&A] [answered]
The title explains it all,  i tried adding the smtp server location, port, of proton, and I get connection refused (no security is enabled)
I re-read the docs and it turns out I need to configure some stuff in proton, I'm assuming as I see you need to configure some stuff in google and so on, how do I configure this in proton though? the documentation doesn't have anything on this.

**Comment by stefan0xC** [comment, marked as answer]
The [first google result when searching for proton smtp](https://proton.me/support/imap-smtp-and-pop3-setup) says this:

> If you’re subscribed to one of our business plans, you can also [set up SMTP on your business applications or devices](https://proton.me/support/smtp-submission) to send emails with Proton Mail.

So if you have a business plan you can follow the linked instructions.
